_FIPS201/introduction.md

```diff
@@ -100,9 +100,9 @@ authentication (i.e., using the PKI-CAK authentication mechanism).
 A backward incompatible change is a change or modification to an existing feature such that the
 modified feature cannot be used with existing relying systems. For example, changing the format of the
 biometric data records would not be compatible with the existing system because a biometric authentication
-attempt with the modified format would fail. Similarly, all systems interacting with the 
+attempt with the modified format would fail. Similarly, since all systems interacting with the 
```

Comment:
I would not make the suggested change.  It reads strange.

Reply:
Can't do this as a suggestion, but what about:

"Similarly, changing the PIV Card Application Identifier (AID) changed (defined in [SP 800-73]) would be a backward incompatible change because all systems interacting with the PIV Card would need to to be modified to use the use AID."